src/sentry/event_manager.py

```diff
@@ -1082,12 +1083,14 @@ def _eventstream_insert_many(jobs):
             is_new = False
             is_regression = False
             is_new_group_environment = False
+            group_category = GroupCategory.ERROR
```

Comment:
Is this absolutely needed? In other places there's the event type property that's used extensively to switch between the error vs transaction processing / post processing. This feels like extra data we are adding to every event for no reason that then gets passed through many steps.

Reply:
We don't want to rely on event level data to understand this, it's a group level property. We don't have any specific plans for this, but there's no reason that a different issue type couldn't be powered by transactions, if we found a valid use case.

Reply:
Ok

Reply:
But this serves no purpose today except as a tool to select a queue anyway. That doesn't seem like a group level thing as events without groups still go through that processing.

Also, isn't this being set anyway on events without a group here?